Makefile

```diff
@@ -7,15 +7,12 @@ endif
 
 current_dir := $(shell basename $(CURDIR))
 
-run:
-	uv run fastapi run
+build:
```

Comment:
I moved `build` to be the first rule because it feels more like the better default. In Socorro and friends, I had the default rule be "help" which listed all the rules and what they did. I like the ergonomics of having the default be "help", but I didn't change that here since it's not a big big deal.

If you prefer having `run` being the default rule, I can change it back. Plus we can adjust things so that if the image isn't built, it builds and then runs.

Do you have a preference for this?

Reply:
no preference!